### PR TITLE
Added gh action for hacktoberfest comment

### PR DESCRIPTION
**Pull request by AvineshTripathi**

### CODE REVIEW

File: .github/workflows/hacktoberfest-comments.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+      - name: Comment on PR
+        if: ${{ github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'hacktoberfest-accepted') }}
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          token: ${{ secrets.GH_ACTIONS }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            A heartfelt thank you, @${{ github.event.pull_request.user.login }}, for your enthusiastic involvement in Hacktoberfest! To express our gratitude, we've crafted a special virtual badge just for you. Don't hesitate to showcase it on your social media profiles and share the love by mentioning Komiser in your posts. Your contribution means the world to us!
+            ![Image](assets/gh/image.png)
```
Comment (mlabouardy) [inline .github/workflows/hacktoberfest-comments.yaml:20]: @AvineshTripathi the image is missing, no?

**Review by mlabouardy — CHANGES_REQUESTED** [review]

File: .github/workflows/hacktoberfest-comments.yaml (hunk 1 above)
Comment (AvineshTripathi) [reply .github/workflows/hacktoberfest-comments.yaml:20]: yep, I couldn't find it so! Also now do you think it is needed? we can create a custom label for crucial prs and give them badges

Comment (mlabouardy) [reply .github/workflows/hacktoberfest-comments.yaml:20]: here's the image:
Hacktoberfest badge
can you add it to the PR @AvineshTripathi

Comment (AvineshTripathi) [reply .github/workflows/hacktoberfest-comments.yaml:20]: where should we keep it? path

**Pull request closed without merging**